src/assets/scripts/client/airline/flightNumberBuilder.js

```diff
@@ -0,0 +1,51 @@
+import { choose } from '../utilities/generalUtilities';
+
+const ALPHA = 'abcdefghijklmnopqrstuvwxyz';
+const NUMERIC = '0123456789';
+
+/**
+ * This picks a random number. If it is the first value within the callsign (ie. i === 0), then it picks a
+ * number between 1 and 9. Otherwise, it picks a number between 0 and 9.
+*/
+const generateNumericValue = (i) => {
```

Comment:
this should be a private function: `_generateNumericValue`

be sure to also add:
```
@function
@param
@return
```
and applicable data types to the docblock